core/templates/create_campaign.html

```diff
@@ -0,0 +1,47 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailimages_tags wagtailadmin_tags static i18n l10n methods_tags %}
+{% block titletag %}{% blocktrans %}Creating campaign{% endblocktrans %}{% endblock %}
+{% block extra_css %}
+    {{ block.super }}
+
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+
+    {% url 'wagtailadmin_tag_autocomplete' as autocomplete_url %}
+    <script>
+        $(function() {
+            $('#id_tags').tagit({
+                autocomplete: {source: "{{ autocomplete_url|addslashes }}"}
+            });
+        });
+    </script>
+
+{% endblock %}
+
+{% block content %}
+    {% trans "Creating Campaign" as editing_str %}
+    {% include "wagtailadmin/shared/header.html" with title=editing_str subtitle=image.title icon="form" %}
+
+    <form action="/admin/create-campaign/" method="POST">
+        {% csrf_token %}
+        <div class="row row-flush nice-padding">
+            <div class="col6">
+                <ul class="fields">
+                    <li>
+                        <label class="label no-float">Page title:</label>
```

Comment:
Thinking about it this label might make more sense as 'Campaign name'.

Reply:
Changed this.